test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -231,14 +231,16 @@ def click(self, element):
                     element.click()
                 failure = None
                 break
-            except Exception as e:
+            except WebDriverException as e:
                 failure = e
                 pass
             try:
                 element = usedfunction() if usedfunction else element
                 self.everything_loaded(element)
```

Comment:
maybe we can remove this line, because it calls that ``data-loaded`` what does the ``sleep``. explicit sleep what you introduced is better.

Reply:
PR #13059 removes it. I didn't do it here, as it would collide with the other PR, and it's also an unrelated change.

Reply:
Makes sense